src/components/TextLink.js

```diff
@@ -32,19 +32,34 @@ const defaultProps = {
 
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
+
+    /**
+     * Checking Function on open TextLink identify whether Clicked or Entered key press
```

Comment:
Now that `event.key && event.type` have been removed from this function I this this comment can be removed also, the code is self-explanatory